app/services/to_cocina/work/mapper.rb

```diff
@@ -55,13 +55,6 @@ def object_type
       end
 
       def document?
-        return false if content.content_files.empty?
-
-        # All files are PDF or hidden
-        return false unless content.content_files.all? do |content_file|
-          content_file.hidden? || content_file.pdf?
-        end
```

Comment:
@justinlittman Was there a reason we needed these checks too?

Reply:
It shouldn't be the document type if there is file that is not hidden and not a PDF.

Reply:
@justinlittman Doesn't line 59 cover that? It returns `true` if there are any content files that are both not hidden and PDF. A not hidden not PDF would cause it to return `false`.

Reply:
Suppose 2 files:
1. Non-hidden PDF
2. Non-hidden JPEG

This should NOT be a document because of 2. However, L59 would be true because of 1.

Reply:
This seems to indicate that there is a missing test: https://github.com/sul-dlss/hungry-hungry-hippo/pull/789

Reply:
Good call, thanks for adding a test

Reply:
@justinlittman Take another look?